moto/cloudwatch/models.py

```diff
@@ -44,18 +60,127 @@ def __init__(self, namespace, name, value, dimensions):
                                      'value']) for dimension in dimensions]
 
 
+class MetricDatumT(BaseModel):
+
+    def __init__(self, namespace, data):
+        self.namespace = namespace
+        self.data = data
+
+
+class MetricDatumv2(BaseModel):
+
+    def __init__(self, namespace, metricdata):
+        self.Namespace = namespace
+    # parse the metricdata object
+
+        self.MetricData = [
+            Metric(
+                MetricName=metric['MetricName'],
+                Dimensions=metric['Dimensions'],
+                Timestamp=metric.get('Timestamp', None),
+                Value=metric.get('Value', None),
+                StatisticValues=metric.get(
+                    'StatisticValues', None),
+                StorageResolution=metric.get(
+                    'StorageResolution', None),
+                Unit=metric.get('Unit', None),)
+            for metric in metricdata]
+
+    def __str__(self):
+        return "{} {}".format(self.Namespace, self.MetricData)
+
+
+class Metric():
+    def __init__(
+            self,
+            MetricName,
```

Comment:
Same here for the variable names. In the boto3 api the keyword arguments to many of the API calls are camel cased but it's pretty unusual in Python and you won't find it in the Moto code base generally.